parts/k8s/addons/kubernetesmasteraddons-azure-npm-daemonset.yaml

```diff
@@ -76,7 +76,7 @@ spec:
         beta.kubernetes.io/os: linux
       containers:
         - name: azure-npm
-          image: containernetworking/azure-npm:v1.0.13
+          image: containernetworking/azure-npm:v1.0.18-rc3

```

Comment:
as @CecileRobertMichon mentioned, we don't want to replace a final release with a newer RC release

Reply:
OK. I will push a final release today and update this.